core/src/mindustry/world/meta/StatValues.java

```diff
@@ -390,7 +390,7 @@ public static <T extends UnlockableContent> StatValue ammo(ObjectMap<T, BulletTy
                     }
 
                     if(type.status != StatusEffects.none){
-                        sep(bt, (type.status.minfo.mod == null ? type.status.emoji() : "") + "[stat]" + type.status.localizedName);
+                        sep(bt, (type.status.minfo.mod == null ? type.status.emoji() : "") + "[stat]" + type.status.localizedName + "[] ~ [stat]" + ((int) type.statusDuration / 60f) + "[] " + Core.bundle.get("unit.seconds"));
```

Comment:
This is still incorrect, you're casting the float to an int, *then* dividing, which converts it back to a float. There's also an unnecessary space after the cast.